### ISSUE TITLE
Linkedin share issue

### ISSUE THREAD

**Issue by Kamleshpaul** [brendt/rfc-vote#243]
https://github.com/brendt/rfc-vote/blob/98217934b0db4b2e8ed4824d3fe579a43a5c9062/resources/views/components/argument-card/share-links.blade.php#L42

i have fix this so many time and and code get overwrite IDK what happen

this is fix of this https://github.com/brendt/rfc-vote/pull/179

**Comment by Plytas** [comment]
Looks like your PR for README https://github.com/brendt/rfc-vote/pull/204 had dirty history. It included a this change and then a revert of it and it was merged after #179 was merged.

---

You should always keep your fork up to date:
<img width="916" alt="image" src="https://github.com/brendt/rfc-vote/assets/17316322/6d098e24-a2dc-4ca8-a533-31a757fec817">

And when you create a PR, always do it from a new branch from `main`.

At this point you'll have to re-submit a new PR with the changes you did in #179

**Comment by Kamleshpaul** [comment]
okay so let me re-submit then thanks @Plytas
